src/utils/inputFilter.ts

```diff
@@ -0,0 +1,17 @@
+export const filterInput = (value: string, allowNegative: boolean, allowDecimal = false) => {
```

Comment:
To maintain consistency, the function name and the file should have the same name. I propose `filterInputValue` to highlight the fact that we want to filter the value.